SU-HSDO/suhumsci
# HSD8-513 added color background option to text area paragraph

# READY FOR REVIEW

# Summary
- Add color field to text area paragraph for background color

# Needed By (Date)
- soon

# Urgency
- low

# Steps to Test

1. checkout this branch
1. sync a site `blt drupal:sync --site=[sitename]`
1. create/edit a basic page
1. add a text area paragraph
1. enter some text or stuff and choose a background color from the options
1. verify the text area paragraph has the appropriate background colors.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5
